src/helix-ui/elements/_hx-element.less

```diff
@@ -2,11 +2,7 @@
 
 * {
   box-sizing: border-box;
-
-  // Text styles need passed through
   color: inherit;
-  font-family: inherit;
-  font-size: inherit;
+  font: inherit;
```

Comment:
I discovered that the `font` shorthand property will set `font-size`, `line-height`, and `font-family` properties previously used.